lib/herd/cluster.ex

```diff
@@ -62,6 +62,20 @@ defmodule Herd.Cluster do
 
       def get_nodes(keys), do: get_nodes(@table_name, @router, keys)
 
+      def add_node(node), do: GenServer.call(__MODULE__, {:add_node, node})
```

Comment:
Small nit: there is a function in Elixir `node/0` that could lead to a bug if this parameter were to be renamed because the `node` in the tuple would still be valid.  It might worth renaming these variables, but I am not sure what a better name would be :p